### PR TITLE
Sticky component - CSS ONLY

### PR DESCRIPTION
**Pull request by mikemai2awesome**
Correct PR for the sticky component. No conflicts with develop.

### CODE REVIEW

**Comment by sghoweri** [comment]
@mikemai2awesome could you update this to include a custom element wrapper (with the class name on this)?

File: src/_patterns/02-components/bolt-sticky/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Getting Started
+The best way to start using Bolt is via the *Bolt Starterkit* (coming soon), a pre-assembled front-end boilerplate designed to get you up and running as quickly as possible.
+
+You can also use the full [Bolt Core](https://www.npmjs.com/package/@bolt/core) CSS framework on its own, or, install just the parts you need and fold the different ITCSS layers of Bolt into your existing codebase.
+
+Installing Bolt’s Action Blocks component is as easy as running NPM install:
+
+```
+npm install @bolt/sticky
```
Comment (sghoweri) [inline src/_patterns/02-components/bolt-sticky/README.md]: @mikemai2awesome @remydenton shouldn't this be `npm install @bolt/components-sticky` to match up with the `name` key in this component's package.json file?

Comment (mikemai2awesome) [reply src/_patterns/02-components/bolt-sticky/README.md]: d'oh. i will have to update all the other components' readme.

**Comment by remydenton** [comment]
Closing in favor of #214, which is a rebased version for publishing.

**Pull request closed without merging**